core/sqlite3_ondisk.rs

```diff
@@ -35,8 +35,9 @@ use std::rc::Rc;
 
 /// The size of the database header in bytes.
 pub const DATABASE_HEADER_SIZE: usize = 100;
+const DEFAULT_CACHE_SIZE: i32 = -2000;
```

Comment:
The default page cache size is indeed 2000 pages in SQLite, but why is this a negative number?

Reply:
> The` default suggested cache size is -2000, which means the cache size is limited to 2048000 bytes of memory 

[source](https://www.sqlite.org/pragma.html#:~:text=Query%20or%20change%20the%20suggested,cache%20size%20of%200%20pages.)

Reply:
@pereman2 Oh, I guess you learn something new every day. Thanks for the clarification!